Homeworks/module13.py

```diff
@@ -0,0 +1,39 @@
+import time
+
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+# Credentials
```

Comment:
For this task it's ok, but I just want to make a reminder for the future that it's not a good practice to store credentials in repo :)